activerecord/lib/arel.rb

```diff
@@ -35,6 +35,9 @@ module Arel
   # Great caution should be taken to avoid SQL injection vulnerabilities.
   # This method should not be used with unsafe values such as request
   # parameters or model attributes.
+  #
+  # Take a look at the {security guide}[https://edgeguides.rubyonrails.org/security.html#sql-injection]
```

Comment:
I didn't wrap this because IMO this looks worse then a line that is 20 characters over the 80 "limit".

```
# Take a look at the {security
# guide}[...] for more
# information
```